<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="Test"
           tests="3"
           failures="1"
           errors="0"
           skipped="0"
           time="34.988">
  <properties />
            <testcase name="ThirdTest" time="0.003" classname="com.cloudleaf.webautomation.CLLoginTest">
                <failure type="java.lang.AssertionError"
                  message="expected [false] but found [true]"
              >
<![CDATA[java.lang.AssertionError: expected [false] but found [true]
  org.testng.Assert.fail(Assert.java:94)
  org.testng.Assert.failNotEquals(Assert.java:494)
  org.testng.Assert.assertEquals(Assert.java:123)
  org.testng.Assert.assertEquals(Assert.java:286)
  org.testng.Assert.assertEquals(Assert.java:296)
  com.cloudleaf.webautomation.CLLoginTest.ThirdTest(CLLoginTest.java:214)
  sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
  sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
  sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
  java.lang.reflect.Method.invoke(Unknown Source)
  org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:80)
  org.testng.internal.Invoker.invokeMethod(Invoker.java:714)
  org.testng.internal.Invoker.invokeTestMethod(Invoker.java:901)
  org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1231)
  org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:127)
  org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
  org.testng.TestRunner.privateRun(TestRunner.java:767)
  org.testng.TestRunner.run(TestRunner.java:617)
  org.testng.SuiteRunner.runTest(SuiteRunner.java:334)
  org.testng.SuiteRunner.runSequentially(SuiteRunner.java:329)
  org.testng.SuiteRunner.privateRun(SuiteRunner.java:291)
  org.testng.SuiteRunner.run(SuiteRunner.java:240)
  org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
  org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
  org.testng.TestNG.runSuitesSequentially(TestNG.java:1198)
  org.testng.TestNG.runSuitesLocally(TestNG.java:1123)
  org.testng.TestNG.run(TestNG.java:1031)
  org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
  org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
  org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
      </failure>

          </testcase>
  
  
            <testcase name="sencodTest" time="0.002" classname="com.cloudleaf.webautomation.CLLoginTest" />
                <testcase name="CloudLeafLogin" time="34.983" classname="com.cloudleaf.webautomation.CLLoginTest" />
      
</testsuite>
